sickbeard/postProcessor.py

```diff
@@ -957,6 +958,15 @@ def process(self):
                 else:
                     cur_ep.release_name = ""
 
+                if cur_ep.torrent_hash != '':
+                    client = clients.getClientIstance(sickbeard.TORRENT_METHOD)()
```

Comment:
because if for any reason the value cur_ep.torrent_hash i don't want to call transmission